test/Governance.test.ts

```diff
@@ -36,6 +36,7 @@ describe('Governance', function () {
     this.delegatee2 = this.signers[3]
     this.delegatee3 = this.signers[4]
     this.newDelegatee = this.signers[5]
+    this.MockVotesOracle = await ethers.getContractFactory('MockVotesOracle')
```

Comment:
suggestion: rename variable as`votesOracle`, as that is the abstract concept being dealt with / tested against.

Reply:
Maybe you meant variable below? 'mockVotesOracle'. This one is just forwards contract factory instance.

Reply:
true, the one below 😁